plugins/@grouparoo/snowflake/__tests__/query-import/import-property.ts

```diff
@@ -50,43 +50,43 @@ describe("snowflake/query/recordProperty", () => {
   });
 
   test("can run a integer query to get a string", async () => {
-    const sql = "SELECT first_name FROM profiles WHERE id = {{ userId }}";
+    const sql = "SELECT first_name FROM profiles WHERE id = {{{ userId }}}";
     const value = await getPropertyValue(sql);
     expect(value).toEqual(["Erie"]);
   });
 
   test("can run a integer query to get a float", async () => {
-    const sql = "SELECT ltv FROM profiles WHERE id = {{ userId }}";
+    const sql = "SELECT ltv FROM profiles WHERE id = {{{ userId }}}";
```

Comment:
I'm curious - would these queries have still worked with the `{{}}`?

Reply:
It does!  Just checked.  Core converts it all correctly and figures it out.